File: src/lib/pjecalc/core/comum/validators/valid-rule.ts
/**
 * PJe-Calc v2.15.1 — ValidRule (interface)
 * Porte de: br.jus.trt8.pjecalc.negocio.comum.validators.ValidRule
 */
import type { Mensagens } from '../mensagens';
import type { ValidValueValidator } from './valid-value-validator';
import type { ValidatorContext } from './validator-context';

export interface ValidRule {
  getMessage(): Mensagens | null;
  isValid(validator: ValidValueValidator, context: ValidatorContext): boolean;
}

File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/comum/validators/ValidRule.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.comum.validators;

import br.jus.trt8.pjecalc.base.comum.validadores.ValidatorContext;
import br.jus.trt8.pjecalc.negocio.comum.validators.ValidValueValidator;
import br.jus.trt8.pjecalc.negocio.constantes.Mensagens;

public interface ValidRule {
    public Mensagens getMessage();

    public boolean isValid(ValidValueValidator var1, ValidatorContext var2);
}

